### PR TITLE
Added some content to packaging guidelines

### PR DESCRIPTION
**Pull request by amoralej**

### CODE REVIEW

File: source/documentation/rdo-packaging-guidelines.html.md [hunk 1]
```diff
@@ -78,6 +112,37 @@ oslo-config-generator --config-file=config-generator/keystone.conf
 
 * Configuration files must be in /etc and not /usr/etc.
 
+### Tests packaging
+
+OpenStack projects provide different tests including unit tests and functional
+tests,  typically using the tempest framework.
```
Comment (hguemar) [inline source/documentation/rdo-packaging-guidelines.html.md]: nitpick: too much spaces after the comma.

File: source/documentation/rdo-packaging-guidelines.html.md [hunk 2]
```diff
@@ -78,6 +112,37 @@ oslo-config-generator --config-file=config-generator/keystone.conf
...
+
+### Other considerations
+
+* To enforce consistency accross OpenStack services packages, use the following snippet to set upstream project name.
+
+```bash
+%global service keystone
+```
+
+Depending on the package type (OpenStack service, library, client, etc...) There
```
Comment (hguemar) [inline source/documentation/rdo-packaging-guidelines.html.md]: I'd just drop the different variables part and emphasize that we provide different example specs to match the different types of projects packaged.

File: source/documentation/rdo-packaging-guidelines.html.md [hunk 3]
```diff
@@ -48,8 +55,22 @@ on the ticket.
...
+Requires=openvswitch.service
+```
+
+* The systemd package provides a set of rpm macros to handle systemd operations
+on %post, %preun and %postun (more details [here](https://fedoraproject.org/wiki/Packaging:Scriptlets#Systemd)).
+
+### python packaging guidelines
 
-### General guidelines
+* Some recommendations about python packaging can be found in
```
Comment (hguemar) [inline source/documentation/rdo-packaging-guidelines.html.md]: Redundant w/ the above statement that we are based upon Fedora packaging guidelines, the goal of this document is to provide specific RDO guidelines or exceptions to the Fedora ones (which are themselves the foundation of RHEL's guidelines)

File: source/documentation/rdo-packaging-guidelines.html.md (hunk 1 above)
Comment (amoralej) [inline source/documentation/rdo-packaging-guidelines.html.md]: fixed.

File: source/documentation/rdo-packaging-guidelines.html.md (hunk 3 above)
Comment (amoralej) [inline source/documentation/rdo-packaging-guidelines.html.md]: removed

File: source/documentation/rdo-packaging-guidelines.html.md (hunk 2 above)
Comment (amoralej) [inline source/documentation/rdo-packaging-guidelines.html.md]: done

**Pull request merged**